mods/common/fluent/chrome.ftl

```diff
@@ -333,9 +333,9 @@ button-settings-title = Settings
 
 ## map-chooser.yaml
 label-mapchooser-panel-title = Choose Map
-button-mapchooser-panel-system-maps-tab = Official Maps
-button-mapchooser-panel-remote-maps-tab = Server Maps
-button-mapchooser-panel-user-maps-tab = Custom Maps
+button-mapchooser-system-maps-tab = Official Maps
+button-mapchooser-remote-maps-tab = Server Maps
+button-mapchooser-user-maps-tab = Custom Maps
```

Comment:
these should be moved outside of chrome.ftl

Reply:
Where and why?

Reply:
because they are no longer referenced in chrome yaml, they should be moved to common.ftl according to the current ftl structure